patches/chrome-installer-mini_installer-chrome.release.patch

```diff
@@ -31,7 +31,14 @@ index 948fc15a146b85f8e4b91e677e54ac2ae77bbabe..8debc2f85562a462ecdf1a1e1517bbc9
  chrome_elf.dll: %(VersionDir)s\
  chrome_pwa_launcher.exe: %(VersionDir)s\
  chrome_wer.dll: %(VersionDir)s\
-@@ -60,6 +70,7 @@ MEIPreload\preloaded_data.pb: %(VersionDir)s\MEIPreload\
+ d3dcompiler_47.dll: %(VersionDir)s\
+ eventlog_provider.dll: %(VersionDir)s\
++gkboaolpopklhgplhaaiboijnklogmbc\list_catalog.json: %(VersionDir)s\gkboaolpopklhgplhaaiboijnklogmbc\
```

Comment:
does every file have to be listed individually here? That doesn't seem right, is this what upstream has for components they ship with?